### PR TITLE
Header component

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by lorenzo-w — DISMISSED** [review]
Looks queer as hell. Love it!

File: src/pages/index.tsx [hunk 1]
```diff
@@ -4,10 +4,12 @@ import Typography from "@material-ui/core/Typography";
 import Box from "@material-ui/core/Box";
 import Link from "@/components/Link";
 import Copyright from "@/components/Copyright";
```
Comment (lorenzo-w) [inline src/pages/index.tsx:6]:
```suggestion
```
This import is unused. Vercel won't let us deploy like that. Removed it for ya 😉

**Review by lorenzo-w — CHANGES_REQUESTED** [review]
Looks good aside from one unused import.

File: src/pages/index.tsx (hunk 1 above)
Comment (thsmdt) [reply src/pages/index.tsx:6]: Actually the import is still needed, as there is a reference to `<Copyright>` further down the file in this revision.
However, as this branch is behind main as of two hours ago. They change you expect will be reflected on a rebase.

**Comment by thsmdt** [comment]
As discussed, I will merge the PR :)

**Pull request merged**